job/runner.go

```diff
@@ -79,3 +91,14 @@ func (j *Job) Run() error {
 	cancel()
 	return err
 }
+
+// Run a command and clean up jobs and pods.
```

Comment:
__[Golint]__ comment on exported method Job.RunAndCleanup should be of the form "RunAndCleanup ..." ([link](https://sider.review/gh/repos/169710725/pulls/25#issue-4259316))

<sub>posted by Sider</sub>